src/silo/common/bidirectional_map.cpp

```diff
@@ -0,0 +1,42 @@
+#include "silo/common/bidirectional_map.h"
+
+#include <optional>
+
+#include "silo/common/pango_lineage.h"
+#include "silo/common/string.h"
+
+namespace silo::common {
+
+template <typename V>
+BidirectionalMap<V>::BidirectionalMap()
+    : id_to_value(),
+      value_to_id() {}
+
+template <typename V>
+V BidirectionalMap<V>::getValue(Idx idx) const {
+   return id_to_value.at(idx);
+}
```

Comment:
Why does `getId` return an optional and this returns the value itself? I'd expect both to return an optional.

Reply:
The way I thought about that:
The `Idx` is internal and if an `Idx` is obtained during the execution of the user query in any way, it must be valid and stay valid indefinitely